index.html

```diff
@@ -37,6 +37,128 @@ <h1 class="name">
 			<div class="bottom_geometry"></div>
 		</section>
 
+		<section class="project_section">
+			<h1 class="projects_label">
+				Projects
+			</h1>
+			<div class="project_card">
+				<div class="project_image"></div>
+				<div class="project_details_card">
+					<p class="project_name">
+						Bavon's Portfolio
+					</p>
+					<div class="project_tecs_container">
+						<div class="tech_card">
+							<p class="tech_text">
+								HTML/CSS
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								Ruby on Rails
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								JavaScript
+							</p>
+						</div>
+					</div>
+					<div class="see_project">
+						<p class="see_project_text">
+							See this project
+						</p>
+						<div class="arrow_forward"></div>
+					</div>
+				</div>
+			</div>
+			<div class="project_card">
+				<div class="project_image_1"></div>
+				<div class="project_details_card">
+					<p class="project_name">
+						Bavon's Portfolio
+					</p>
+					<div class="project_tecs_container">
+						<div class="tech_card">
+							<p class="tech_text">
+								HTML/CSS
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								Ruby on Rails
+							</p>
+						</div>
+						<div class="tech_card">
+							<p class="tech_text">
+								JavaScript
+							</p>
+						</div>
+					</div>
+					<div class="see_project">
+						<p class="see_project_text">
+							See this project
+						</p>
+						<div class="arrow_forward"></div>
+					</div>
+				</div>
+			</div>
+		</section>
+		<section class="about_section">
```

Comment:
- [ ] About myself section width has to same with project section. Please fix it.

![image](https://user-images.githubusercontent.com/38635158/148407448-9b2efe53-633b-45d8-a31f-46e5b525b884.png)
___
### ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/148407567-938cb632-16e5-4983-b1fc-5aa5b87fa7a7.png)